src/components/film-filter/film-filter.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import {
+  Row, Col, Dropdown, DropdownToggle,
+} from 'reactstrap';
+import PropTypes from 'prop-types';
+
+import { useFilters } from '../../hooks/use-filters';
+import { DropDownMenu } from '../drop-down-menu/drop-down-menu';
+import { IconArrow } from '../icons/icon-arrow';
+
+export const FilmFilter = ({
+  items, defaultValue, filterName,
+}) => {
+  const { dispatchAddNewFilter, dispatchRemoveFilter, filters } = useFilters();
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+
+  const getFilterValue = () => {
+    const selectedItem = filters.find(filter => filter.type === filterName);
+
+    if (selectedItem) {
+      return selectedItem.name;
+    }
+
+    return defaultValue;
+  };
+
+  const selectItem = ({ value, name }) => {
+    if (value === 'ALL') {
+      dispatchRemoveFilter({ value, type: filterName });
```

Comment:
do you really need a value to remove filter?

Reply:
done